spring-petclinic-microservices/spring-petclinic-customers-service/src/main/resources/application.properties

```diff
@@ -1,17 +1,10 @@
-azure.keyvault.enabled=true
-azure.keyvault.uri=${AZURE_KEYVAULT_URI}
-azure.keyvault.client-id=${AZURE_KEYVAULT_CLIENTID}
-azure.keyvault.client-key=${AZURE_KEYVAULT_CLIENTKEY}
-azure.keyvault.tenant-id=${AZURE_KEYVAULT_TENANTID}
-azure.keyvault.token-acquire-timeout-seconds=60
-azure.keyvault.refresh-interval=1800000
-azure.keyvault.secret-keys=cosmosdburi,cosmosdbkey,cosmosdbsecondarykey, redisuri, redispassword
+spring.cloud.bootstrap.enabled=true
```

Comment:
why both application.properties and  application.yaml

Reply:
the code use @propertysource,it doesn't support yaml by default, and I prefer to use peoperties. Yes, it's strange here, I will update it.